src/Workspaces/Core/Desktop/Workspace/Storage/PersistentStorageService.cs

```diff
@@ -112,7 +112,7 @@ private IPersistentStorage GetStorage(Solution solution, string workingFolderPat
                     // everything seems right, use what we have
                     if (storage?.WorkingFolderPath == workingFolderPath)
                     {
-                        storage.AddRefUnsafe();
+                        storage?.AddRefUnsafe();
```

Comment:
@jasonmalinowski, I agree, the `PersistentStorageService.GetStorage` method is called from one place only, and there's a check that `workingFolderPath` is not null, so probably this check is excessive. However, if someone will pass a null as a `workingFolderPath` argument, it could cause a NullReferenceException,